hat/assets/js/apps/Iaso/domains/app/translations/en.json

```diff
@@ -793,42 +793,42 @@
     "iaso.pages.title": "Pages",
     "iaso.periods.end": "End period",
     "iaso.periods.start": "Start period",
-    "iaso.permission.reports": "Reports",
-    "iaso.permissions.assignments": "Assignments",
-    "iaso.permissions.completeness": "Data completeness",
-    "iaso.permissions.completeness_stats": "Completeness stats",
-    "iaso.permissions.dataTasks": "Tasks",
-    "iaso.permissions.entities": "Beneficiaries",
-    "iaso.permissions.forms": "Forms",
+    "iaso.permission.reports": "Devices",
```

Comment:
"Reports" is "devices" ???

Reply:
![Screenshot from 2023-09-22 15-45-12](https://github.com/BLSQ/iaso/assets/19631540/c3ab728d-5ab5-4df6-9eab-e038ca2654e1)